app/src/main/java/com/example/atheneum/fragments/AddBookFragment.java

```diff
@@ -99,7 +103,7 @@ public void populateFieldsByIsbn() {
         // TODO: FINISH
 
         Log.i("AddBook", "AddBook*** Auto-populate requested");
-        long isbn = -1;
+        this.isbn = -1;
```

Comment:
If we use -1 as an invalid ISBN for the rest of our app that would be a good integer constant to put in the book class.

Reply:
Done. I added a static constant to Book.